cli/src/main/resources/help/module.txt

```diff
@@ -3,7 +3,7 @@ SYNOPSIS
   To add a module locally
 
     module add [--module-root-dir=modules_root_directory_path] --name=module_name 
-               [--slot=slot_name] --(absolute-)resources=list_of_jars [--resource-delimiter=delimiter]
+               [--slot=slot_name] [--resources=list_of_jars] [--absolute-resources=list_of_jars] [--resource-delimiter=delimiter]
```

Comment:
ok, added new line, it still exceeds 80 characters a bit, but it the line was longer even before my changes